pkg/services/ngalert/notifier/autogen_alertmanager.go

```diff
@@ -77,6 +77,10 @@ func newAutogeneratedRoute[R receiver](ctx context.Context, logger log.Logger, s
 	}
 	newAutogenRoute, err := generateRouteFromSettings(cfg.GetRoute().Receiver, notificationSettings)
 	if err != nil {
+		if skipInvalid {
+			logger.Error("Generation of simplified routing policies failed. Skipping", "error", err)
+			return newAutogenRoute, nil
+		}
```

Comment:
it looks it will fail all autogen routes if at least one is not valid. It would be great if we could skip on 1 level down - contact point level